public/index.html

```diff
@@ -187,10 +195,32 @@
        </div>
     </div>
     <div class="row">
-      <div class="col-lg-12">
+      <div class="col-6">
         <zack-filter-summary id="filter-container" class="zack-filter">&nbsp;
         </zack-filter-summary>
       </div>
+      <div class="col-6" style="text-align: right">
+        <zack-query-order-menu>
+          Order:
+          <zack-query-order id="order-by-score" selected descending variable="totalScore">
```

Comment:
On load any `[selected]` element will announce itself to set the initial sorting